dev_tools/oneapi/convert_source.sh

```diff
@@ -167,8 +167,10 @@ fi
 rm "${OUTPUT_FILE}"
 echo "#define GET_QUEUE 0" >> "${OUTPUT_FILE}"
 # add empty ginkgo license such that format_header recognize some header before header def macro
-echo "/*${GINKGO_LICENSE_BEACON}" >> "${OUTPUT_FILE}"
-echo "${GINKGO_LICENSE_BEACON}*/" >> "${OUTPUT_FILE}"
+CURRENT_YEAR=$(date +%Y)
+echo "${GINKGO_LICENSE_BEGIN} 2017-${CURRENT_YEAR} The Ginkgo authors" > "${OUTPUT_FILE}"
+echo "//" >> "${OUTPUT_FILE}"
+echo "${GINKGO_LICENSE_END} BSD-3-Clause" >> "${OUTPUT_FILE}"
```

Comment:
```suggestion
echo "${GINKGO_LICENSE_BEGIN} 2017-${CURRENT_YEAR} The Ginkgo authors" >> "${OUTPUT_FILE}"
echo "//" >> "${OUTPUT_FILE}"
echo "${GINKGO_LICENSE_END} BSD-3-Clause" >> "${OUTPUT_FILE}"
```
It should be controlled by add_licences.
It's fine if it can be fixed automatically when they are different from the add_licences.sh
need to use append >>

Reply:
Do you mean to remove this part, and just let add_license.sh handle this?

Reply:
No, I think add_license.sh needs `// SPDX` to add the corresponding license?
I would like to ensure the add_license.sh is the final license control to ensure it will be correct even if it is wrong here.
The license is hard-coded in more than one place, so it may be easy to miss one of them in the future

Reply:
The add_license.sh script is getting removed in the next PR anyway, so perhaps we shouldn't dwell on this too long. I will create another PR showing how it will be removed.

Reply:
BTW, here is the PR that removes the script: #1460